starlette/responses.py

```diff
@@ -12,6 +11,7 @@
 
 import anyio
 
+from starlette._crypto import md5_hexdigest
```

Comment:
I made this a "private" module. I know there isn't much history of this in Starlette, but I figure this is as good as any other use case to start with.

Reply:
I think it's fine. Although I think we should use `_compat.py` to be aligned with `httpx` and `httpcore`

Reply:
done